plugins/extractors/maxcompute/maxcompute.go

```diff
@@ -0,0 +1,250 @@
+package maxcompute
+
+import (
+	"context"
+	_ "embed" // used to print the embedded assets
+	"fmt"
+	"sync"
+
+	"github.com/aliyun/aliyun-odps-go-sdk/odps"
+	"github.com/aliyun/aliyun-odps-go-sdk/odps/account"
+	"github.com/goto/meteor/models"
+	v1beta2 "github.com/goto/meteor/models/gotocompany/assets/v1beta2"
+	"github.com/goto/meteor/plugins"
+	"github.com/goto/meteor/registry"
+	"github.com/goto/meteor/utils"
+	"github.com/goto/salt/log"
+	"google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type Config struct {
+	ProjectName     string `mapstructure:"project_name"`
+	EndpointProject string `mapstructure:"endpoint_project"`
+	AccessKey       struct {
+		ID     string `mapstructure:"id"`
+		Secret string `mapstructure:"secret"`
+	} `mapstructure:"access_key"`
+	SchemaName     string `mapstructure:"schema_name,omitempty"`
+	MaxPreviewRows int    `mapstructure:"max_preview_rows,omitempty"`
+	Exclude        struct {
+		Schemas []string `mapstructure:"schemas"`
+		Tables  []string `mapstructure:"tables"`
+	} `mapstructure:"exclude,omitempty"`
+	Concurrency      int  `mapstructure:"concurrency,omitempty"`
+	BuildViewLineage bool `mapstructure:"build_view_lineage,omitempty"`
+	// MaxPageSize      int32 `mapstructure:"max_page_size,omitempty"`
+	// MixValues            bool     `mapstructure:"mix_values,omitempty"`
+	// IncludeColumnProfile bool     `mapstructure:"include_column_profile,omitempty"`
+	// IsCollectTableUsage  bool     `mapstructure:"collect_table_usage,omitempty"`
+	// UsagePeriodInDay     int      `mapstructure:"usage_period_in_day,omitempty"`
+	// UsageProjectNames    []string `mapstructure:"usage_project_names,omitempty"`
+}
+
+type Extractor struct {
+	plugins.BaseExtractor
+	logger log.Logger
+	config Config
+}
+
+//go:embed README.md
+var summary string
+
+var sampleConfig = `
+project_name: goto_test
+endpoint_project: maxcompute.ap-southeast-5.aliyuncs.com
+access_key:
+    id: access_key_id
+    secret: access_key_secret
+schema_name: DEFAULT
+max_preview_rows: 3
```

Comment:
remove this since preview rows won't be supported for first milestone

Reply:
addressed